src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -2140,5 +2140,10 @@ public bool IsImplicitObjectCreation(SyntaxNode node)
             return ((CSharpSyntaxNode)node).Kind() == SyntaxKind.ImplicitObjectCreationExpression;
 #endif
         }
+
+        public SyntaxNode GetExpressionOfThrowExpression(SyntaxNode throwExpression)
+            => ((ThrowExpressionSyntax)throwExpression).Expression;
+
+        public bool IsThrowStatement(SyntaxNode node) => node?.IsKind(SyntaxKind.ThrowStatement) == true;
```

Comment:
```suggestion
        public bool IsThrowStatement(SyntaxNode node) => node.IsKind(SyntaxKind.ThrowStatement);
```

(iskind is null safe).  or:

`node is ThrowExpressionSyntax`.

Reply:
wrap `=>` to next line, indented one level.